src/utils/localStorageOrders.ts

```diff
@@ -0,0 +1,121 @@
+import { Order } from '@gelatonetwork/limit-orders-lib'
+import { get, set, clear } from 'local-storage'
+
+const LS_ORDERS = 'gorders_'
```

Comment:
Can't we use usePersistState ?

Reply:
yea I'll rewrite some of the localStorage logic today, some stuff is not working properly yet, its a draft PR 😅